index.html

```diff
@@ -132,5 +123,251 @@
     
         </section>
     </header>
+
+    <main class="works">
```

Comment:
- [ Required ] It would be better if you try as much as possible to stick with the colors provided on the Figma design, If you look at the background color around the `work-cards` you'll notice that the background color on your design doesn't match the background color on the Figma design. Please see the screenshot below for more details on this.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/221296415-a96f2ae0-e208-4419-ba80-ced9ac886e7d.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/221296654-3942d035-4876-4346-83f5-8d273a340545.png" width="375px" />